test/dotcom_web/channels/predictions_channel_test.exs

```diff
@@ -89,13 +114,15 @@ defmodule DotcomWeb.PredictionsChannelTest do
 
     test "filters out past predictions", context do
       # Setup
-      now = Timex.now() |> Timex.shift(seconds: 1)
-      past = Timex.shift(now, seconds: -15)
-      future = Timex.shift(now, seconds: 15)
+      now = now() |> DateTime.shift(second: 1)
+      past = DateTime.shift(now, second: -15)
+      future = DateTime.shift(now, second: 15)
```

Comment:
Why are we shifting it 1 second and then 15 seconds? I remember something about predictions being off or needing to be rounded. But, if we're just looking for before and after we should shift by the smallest increment. Otherwise, why not 1 minute or 1 hour, etc.

Reply:
That's a great point, why _not_ 1 minute or hour...